posts/2021-05-11-database-ssl-primer.adoc

```diff
@@ -323,9 +336,9 @@ javax.net.ssl|DEBUG|35|Default Executor-thread-3|X509TrustManagerImpl.java:79|ad
 ----
 
```

Comment:
what does the user do to resolve the error once they determine what trusted certificates were added? also- added by whom, to what?

Reply:
I would say the resolution would be to:
"Check your server configuration and ensure that the truststore configured on the JDBC driver, or `transportSecurity-1.0` exists and contains the certificate you expect the database to send."